juju/juju
# Change how offer information is printed in status and offers CLI

## Description of change

The offer information as rendered in juju status and juju offers is changed.

juju status shows a summary of each offer (what used to be printed by juju offers)

juju offers now has 2 tabular formats:
- summary, which shows a one line summary of each offer like in status
- tabular (the default), which shows for each offer who is connected and the relation id

## QA steps

Run up a cmr scenario
Use the CLI and verify output:
juju status
juju offers
juju offers --format summary
juju offers --format yaml
